changefeedtrigger/readme.md

```diff
@@ -0,0 +1,49 @@
+# README
+The purpose of this sample is to create a log of all new events that occur in a storage account with multidimensional filtering that runs on a timer using an Azure Function
+
+## Prerequisites
+An Azure Storage Account
+
+An Timer Trigger Azure Function
+
+## How to Use
+This sample includes an __init__.py file as the main program, a function.json to configure the timer trigger function, and a blobEventsSample.py file that represents a data creation sample by creating events to test the program on.
+
+To use the data creation sample, you must go into the code an add your personal connection string
+
+To use the function, you must go into the __init__ file and add your connection string as well as your container name, and which filters you wish to employ!
+
+## Building An Azure Function
+For help setting up an Azure Function follow these steps:
+
+### Create a Function App
```

Comment:
I added that link at the bottom in case extra help is needed, the only reason i separated it was because the instructions on that link do not work for using Python